rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -835,8 +835,8 @@ protected NodeRunner buildNodeRunner() {
                 getRskSystemProperties().flushNumberOfBlocks(),
                 getCompositeEthereumListener(),
                 getTrieStore(),
-                getBlockStore()
-        ));
+                getBlockStore(),
+                null));
```

Comment:
Shouldn't call to "getReceiptStore()" instead of null?

Reply:
Good catch! Fixing it